lib/bemxjst/index.js

```diff
@@ -302,6 +303,7 @@ BEMXJST.prototype.run = function(json) {
 
   this.match = match;
   this.context = context;
+  this.depth = depth;
```

Comment:
@tadatuta but in previous commit I decided to use `_run()`. Maybe now it’s much better solution…